com.ibm.streamsx.topology/opt/python/include/splpy.h

```diff
@@ -263,6 +322,9 @@ namespace streamsx {
       PyObject * arg = pyAttributeToPyObject(splVal);
 
       // invoke python nested function that generates the int32 hash
+      // clear any indication of an error and then check later for an 
+      // error
+      PyErr_Clear();
```

Comment:
this was put in because I discovered the PyLong_AsLong() doesn't necessarily throw an exception but just sets an error -- which can be tested for later. I print the error and let the 0 hash value get returned.